currency-rates/arch-go.yaml

```diff
@@ -0,0 +1,53 @@
+version: 1
+threshold:
+  compliance: 100
+  coverage: 75
+dependenciesRules:
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
+  - package: "**.clients.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.handlers.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.repo.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.routers.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.handlers.**"
+        - "**.tools.**"
+  - package: "**.scheduler.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.tools.**"
+  - package: "**.services.**"
+    shouldOnlyDependsOn:
+      internal:
+        - "**.repo.**"
+        - "**.tools.**"
```

Comment:
You check services for only internal dependencies, but in `currency-rates/internal/services/mail.go` there are dependencies for external libraries ("crypto/tls",  "net/smtp").

Why do business policies know smth about encryption?

Reply:
Also, what is the benefit of explicit describing of dependencies, when tested packages can depend on any external library?

Reply:
I decided that the arch test should test only my internal architectures. That's why I didn't focus on libs. However, "crypto/tls" and "net/smtp" will be moved from here in a future PR, so for now, let them stay here)